### PR TITLE
Refactor log origin string

### PR DESCRIPTION
**Pull request by andrejro**
Use externally passed variable to validate log origin string.

### CODE REVIEW

**Review by abarisani — APPROVED** [review]

**Review by abarisani — APPROVED** [review]

File: cmd/create_proofbundle/main.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ var (
 	release       = flag.String("release", "armory-drive.release", "Path to release metadata file")
 	logURL        = flag.String("log_url", "https://raw.githubusercontent.com/f-secure-foundry/armory-drive-log/master/log/", "URL identifying the location of the log")
 	logPubKeyFile = flag.String("log_pubkey_file", "", "Path to file containing the log's public key")
-	logOrigin     = flag.String("log_origin", api.OriginV0, "The expected first line of checkpoints issued by the log")
+	logOrigin     = flag.String("log_origin", "", "The expected first line of checkpoints issued by the log")
 	outputFile    = flag.String("output", "", "Path to write output file to, leave unset to write to stdout")
 	timeout       = flag.Duration("timeout", 10*time.Second, "Maximum duration to wait for release to become integrated into the log")
```
Comment (AlCutter) [inline cmd/create_proofbundle/main.go:46]: Might be worth a not empty check in main to ensure it's set?

Comment (andrejro) [reply cmd/create_proofbundle/main.go:46]: Right, added that check.

**Comment by andrejro** [comment]
@AlCutter let me know if it is ok now.

**Comment by andrejro** [comment]
Will merge this to allow us continue setting up the FT log environment. @AlCutter let me know if there is any other issue with this PR.

**Pull request merged**